README.md

```diff
@@ -3,31 +3,112 @@
 [![Python package](https://img.shields.io/pypi/v/promptflow)](https://pypi.org/project/promptflow/)
 [![Python](https://img.shields.io/pypi/pyversions/promptflow.svg?maxAge=2592000)](https://pypi.python.org/pypi/promptflow/) 
 [![PyPI - Downloads](https://img.shields.io/pypi/dm/promptflow)](https://pypi.org/project/promptflow/)
+[![CLI](https://img.shields.io/badge/CLI-reference-blue)](https://microsoft.github.io/promptflow/reference/pf-command-reference.html)
+[![vsc extension](https://img.shields.io/visual-studio-marketplace/i/prompt-flow.prompt-flow?logo=Visual%20Studio&label=Extension%20)](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow)
+
+[![Doc](https://img.shields.io/badge/Doc-online-green)](https://microsoft.github.io/promptflow/index.html)
+[![Issue](https://img.shields.io/github/issues/microsoft/promptflow)](https://github.com/microsoft/promptflow/issues/new/choose)
+[![Discord](https://dcbadge.vercel.app/api/server/bnXr6kxs?compact=true&style=flat)](https://discord.gg/bnXr6kxs)
+[![Discussions](https://img.shields.io/github/discussions/microsoft/promptflow)](https://github.com/microsoft/promptflow/issues/new/choose)
+[![CONTRIBUTING](https://img.shields.io/badge/Contributing-8A2BE2)](https://github.com/microsoft/promptflow/blob/main/CONTRIBUTING.md)
 [![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
 
-> Welcome to join us to make Prompt flow better by
-> participating [discussions](https://github.com/microsoft/promptflow/discussions),
-> opening [issues](https://github.com/microsoft/promptflow/issues/new/choose),
-> submitting [PRs](https://github.com/microsoft/promptflow/pulls).
+> Welcome to join us to make Prompt flow!
```

Comment:
do we intensionally omit the better?

Reply:
all links have been moved to the tags above, so I think there may be no need to show it again, and I even think it would be okay to delete this sentence. However, I don't have a strong opinion and feel free to move them back if you feel they need to be highlighted.

Reply:
I think the sentence is important to keep, as we want to express our attitude that we want to take contribution from the  community. But we might change the form and place as we remove the links to tags.

My question wss just:
Welcome to join us to make Prompt flow!
or 
Welcome to join us to make Prompt flow better!

Reply:
Got it, sorry for misunderstanding before